wa/framework/target/descriptor.py

```diff
@@ -416,26 +416,30 @@ def get_descriptions(self):
     'chromeos': ChromeOsAssistant
 }
 
-# name --> ((platform_class, conn_class), params_list, defaults)
+# name --> (((platform_class, conn_class), params_list, defaults), target_defaults)
 # Note: normally, connection is defined by the Target name, but
 #       platforms may choose to override it
+# Note: the target_defaults allows you to override common target_params for a
+# particular platform. Parameters you can override are in COMMON_TARGET_PARAMS
+import re
```

Comment:
This should not be imported here. Is it needed at all?

Reply:
Sorry. Whoops! Have removed it.